src/Home.js

```diff
@@ -1,9 +1,54 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import data from './data';
+import styles from './css/Home.module.css';
+import Homemap from './Homemap';
 
 function Home() {
+  const [continent, setContinent] = useState('All');
+  const [newdata, setNewdata] = useState(data);
+  useEffect(() => {
+    if (continent === 'All') {
+      setNewdata(data);
+    } else {
+      setNewdata(data.filter((ite) => ite.name === continent));
+    }
+  }, [continent]);
   return (
-    <div>
-      hello home
+    <div className={styles.home_container}>
+      <div className={styles.home_container_world_div}>
+        <figure className={styles.home_container_world_div_figure}>
+          <img src="/assets/world_map.svg" alt="world" />
+        </figure>
+        <h2>
+          The Whole World
+        </h2>
+      </div>
+      <div>
+        <select
+          value={continent}
+          className={styles.home_container_items_select}
+          onChange={(e) => setContinent(e.target.value)}
+        >
+          <option value="All">Select Continent</option>
+          <option value="Africa">Africa</option>
+          <option value="Asia">Asia</option>
+          <option value="Antarctic">Antarctic</option>
+          <option value="Europe">Europe</option>
+          <option value="North America">North America</option>
+          <option value="Oceania">Oceania</option>
+          <option value="South America">South America</option>
+        </select>
+        <div className={continent === 'All' ? styles.home_container_items_div : styles.home_container_items_div2}>
+          {
+            newdata.map((item) => (
+              <Homemap
+                key={item.id}
+                datas={item}
+              />
+            ))
+          }
+        </div>
+      </div>
     </div>
   );
 }
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)for this project,  " ` Home page: When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` ", kindly add a component, e.g a search box that can be used to filter what is displayed on the home page(`Home page has a list of categories that could be filtered using given parameters, e.g. category name.`). For example, you should be able to filter components based on their names.

Reply:
## Thank you
# i have resolved it by including a search box in the home to filter by continent name